Makefile

```diff
@@ -8,7 +8,7 @@ export GOFLAGS=-mod=vendor
 export GO111MODULE=on
 export CGO_ENABLED=0
 export GOOS=linux
-export GOARCH=amd64
+export GOARCH=$(uname -m)
```

Comment:
Maybe I'd just allow overriding it so we can make sure that if e.g. GitHub Actions are run in an ARM machine, they won't break the containers we deploy to Quay.
```Makefile
GOARCH ?= amd64
```